src/probot.ts

```diff
@@ -121,6 +121,6 @@ export class Probot {
       return;
     }
 
-    return appFn(this, {});
+    return await appFn(this, {});
```

Comment:
the `load` function is already async, this change makes no difference. If `appFn` is async it will return a promise, and `await server.load` will await that promise, no need to add an await here

Reply:
Do you mean because a async function will only wrap a return value in a promise if it is not a promise itself?
As described here?: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function#rewriting_a_promise_chain_with_an_async_function

I removed the await here.

Reply:
yes, pretty much 👍🏼